waspc/data/Generator/templates/react-app/src/operations/resources.js

```diff
@@ -31,26 +31,26 @@ export function addResourcesUsedByQuery(queryCacheKey, resources) {
 export function getQueriesUsingResource(resource) {
   return Array.from(resourceToQueryCacheKeys.get(resource) || [])
 }
-
 /**
  * Invalidates all queries that are using specified resources.
  * @param {string[]} resources - Names of resources.
  */
-export function invalidateQueriesUsing(resources) {
-  const queryCacheKeysToInvalidate = new Set()
-  for (const resource of resources) {
-    getQueriesUsingResource(resource).forEach(key => queryCacheKeysToInvalidate.add(key))
-  }
-  for (const queryCacheKey of queryCacheKeysToInvalidate) {
+export async function invalidateQueriesUsing(resources) {
+  const queryClient = await queryClientInitialized
```

Comment:
I think we can close this discucssion since the same topic is covered in [the other thread](https://github.com/wasp-lang/wasp/pull/612#discussion_r886993962).